src/components/grant-details-page/grant-details-sidebar/GrantDetailsSidebar.js

```diff
@@ -1,14 +1,14 @@
 import Link from 'next/link';
 
-export function GrantDetailsSidebar({ grantLabel }) {
+export function GrantDetailsSidebar({ grantLabel, grantId }) {
   return (
     <div className="govuk-grid-column-one-quarter">
       <hr className="govuk-section-break govuk-section-break--visible govuk-!-margin-bottom-2 govuk-border-colour"></hr>
       <h2 className="govuk-heading-m">Get updates about this grant</h2>
       <Link
         href={{
           pathname: '/subscriptions/signup',
-          query: { id: grantLabel },
+          query: { id: grantId, grantLabel: grantLabel },
```

Comment:
ultra minor point but may as well shorthand it
```suggestion
          query: { id: grantId, grantLabel },
```